src/util/theme.py

```
@@ -404,3 +459,22 @@ def icon(
                 selected = self.pixmap(splitExt[0] + "_selected" + splitExt[1], themed=themed)
                 icon.addPixmap(selected, QtGui.QIcon.Mode.Selected, QtGui.QIcon.State.On)
             return icon
+
+    def find_stylesheet_attribute(self, section: str, attribute: str, fallback: str = "") -> str:
+        if (found := self._theme.find_stylesheet_attribute(section, attribute)) is None:
+            return self._default_theme.find_stylesheet_attribute(section, attribute) or fallback
+        else:
+            return found or fallback
+
+    def find_stylesheet_style(self, section: str, fallback: str = "") -> str:
```

Ruff (0.14.13)

Unused method argument: `fallback`

(ARG002)